tests_integration/workspaces/test_validate_schema.py

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2024 Snowflake Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+import os
+from unittest import mock
+
+
+@pytest.mark.integration
+@mock.patch.dict(
+    os.environ,
+    {
+        "SNOWFLAKE_CLI_FEATURES_ENABLE_PROJECT_DEFINITION_V2": os.environ.get(
+            "SNOWFLAKE_CLI_FEATURES_ENABLE_PROJECT_DEFINITION_V2", "true"
+        ),
```

Comment:
Would we ever want to override this to be `false` for this test?

Reply:
To test that v2 is not recognized by default? I have a unit test that covers it.

Reply:
Yeah, that's fair, what I'm wondering is why this isn't just `"SNOWFLAKE_CLI_FEATURES_ENABLE_PROJECT_DEFINITION_V2": "true"`

Reply:
Why would we need to override the test with an env var?